### ISSUE TITLE
Update release notes for cmc (Trac #1972)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1964]
Migrated from https://code.icecube.wisc.edu/ticket/1972
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "", 
    "reporter": "juancarlos", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "Update release notes for cmc", 
    "priority": "blocker", 
    "keywords": "cmc", 
    "time": "2017-03-22T16:51:05", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170324T16:39:52: 
* jvansanten commented:

> In changeset 154231:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="154231"
> Update release notes. Closes #1964.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
